src/VPolygon.jl

```diff
@@ -585,10 +629,10 @@ function minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
     mP = length(vlistP)
     mQ = length(vlistQ)
     i = 1
-    σP = σ(N[1, 0], P)
-    σQ = σ(N[1, 0], Q)
-    k = findfirst(==(σP), vlistP)
-    j = findfirst(==(σQ), vlistQ)
+    σP = _binary_support_vector(N[1, 0], P)
+    σQ = _binary_support_vector(N[1, 0], Q)
+    k = findfirst(==(σP(vlistP)), vlistP)
```

Comment:
`_binary_support_vector(N[1, 0], P)` returns an index, hence `σP` is an index. What does `σP(vlistP)` mean then?